client_auth.go

```diff
@@ -2,6 +2,9 @@ package vnc
 
 import (
 	"net"
+
+	"crypto/des"
+	"encoding/binary"
```

Comment:
Both of these shouldn't have a newline above it. Standard Go style is to group any "stdlib" and separate only external with a newline.